tests/SprykerEcoTest/Zed/Braintree/Business/BraintreeFacade/BraintreeFacadePreCheckTest.php

```diff
@@ -0,0 +1,123 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEcoTest\Zed\Braintree\Business\BraintreeFacade;
+
+use Braintree\Transaction\CreditCardDetails;
+use SprykerEco\Zed\Braintree\BraintreeConfig;
+use SprykerEco\Zed\Braintree\Business\Payment\Transaction\AbstractTransaction;
+use SprykerEco\Zed\Braintree\Business\Payment\Transaction\PreCheckTransaction;
+use SprykerEco\Zed\Braintree\Dependency\Facade\BraintreeToMoneyFacadeBridge;
+use SprykerEco\Zed\Braintree\Dependency\Facade\BraintreeToMoneyFacadeInterface;
+
+/**
+ * Auto-generated group annotations
+ *
+ * @group SprykerEcoTest
+ * @group Zed
+ * @group Braintree
+ * @group Business
+ * @group Facade
+ * @group BraintreeFacadePreCheckTest
+ * Add your own group annotations below this line
+ */
+class BraintreeFacadePreCheckTest extends AbstractFacadeTest
+{
+    /**
+     * @return void
+     */
+    public function testPreCheckPaymentWithSuccessfulResponse(): void
+    {
+        // Arrange
+        $braintreeBusinessFactory = $this->getFactoryMock(['createPreCheckTransaction']);
+        $braintreeBusinessFactory->expects($this->once())->method('createPreCheckTransaction')->willReturn(
+            $this->getPreCheckTransactionMock(),
+        );
+        $braintreeFacade = $this->getBraintreeFacade($braintreeBusinessFactory);
+        $orderTransfer = $this->createOrderTransfer();
+        $quoteTransfer = $this->createQuoteTransfer($orderTransfer);
+
+        // Act
+        $response = $braintreeFacade->preCheckPayment($quoteTransfer);
```

Comment:
```suggestion
        $braintreeTransactionResponseTransfer = $braintreeFacade->preCheckPayment($quoteTransfer);
```

Reply:
don't we need to assert message/code etc?
same comments for similar tests below

Reply:
Added assert messages